### ISSUE TITLE
MA0003 warns even if there's no tuple name

### ISSUE THREAD

**Issue by Piedone** [meziantou/Meziantou.Analyzer#284]
**Context**
Rule identifier: _MA0003_
Target Framework: _.NET 5.0_
Language version: _9.0_

**Describe the bug**

MA0003 warns even if the tuple element has no name and thus ignored.

**To Reproduce**

````c#
public class Test
{
    public (bool, bool) MyMethod() => (myName1: true, myName2: true);
}
````

If `myName1` and `myName2` are not specified then MA0003 will show a violation for them. However, if they are specified we'll get a violation from Microsoft.CodeAnalysis.CSharp:

```
CS8123	The tuple element name 'myName1' is ignored because a different name or no name is specified by the target type '(bool, bool)'.
``` 

This seems to be a language issue. Thus, I think MA0003 shouldn't surface a violation if the tuple element doesn't have a name.

In the above example, we could fix this by specifying element names too but this is not possible if the tuple is part of an API.

**Additional context**

This came up when providing the `map` to [this method](https://github.com/sebastienros/yessql/blob/2c5aeda50402ef09bff1127a42774a85dc6fcdc3/src/YesSql.Filters.Query/QueryUnaryEngineBuilder.cs#L47).

**Comment by meziantou** [comment]
It should be fixed in the version 1.0.686

**Comment by Piedone** [comment]
Works, great, thank you!
